go/api-frontend/aaa/authentication_backend_test.go

```diff
@@ -28,7 +28,7 @@ func TestMemAuthenticationBackend(t *testing.T) {
 		t.Error("User was unauthenticated although it presented valid credentials. error:", err)
 	}
 
-	if !tokenInfo.AdminRoles["SYSTEM_READ"] {
+	if !tokenInfo.AdminRoles()["SYSTEM_READ"] {
```

Comment:
The name AdminRoles is confusing here since in perl there called actions not roles.